Jenkinsfile

```
@@ -12,7 +12,7 @@ gitflow = new GitFlow(this, git)
 github = new GitHub(this, git)
 changelog = new Changelog(this)
 Docker docker = new Docker(this)
-goVersion = "1.18"
+goVersion = "1.20"
```

```suggestion
goVersion = "1.22.5"
```
use the current go-version